firmware/spade/src/pc/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
-target_link_libraries(spade PRIVATE -fno-omit-frame-pointer -fsanitize=undefined)
+set(DIR_TO_BSD_LIB_FILE /usr/lib64/libbsd.so)
```

Comment:
What does this do?  Can you add a comment here explaining why this is needed?  Does this work on all systems?

Reply:
this looks very un-portable

Reply:
removed! was initially a hacky solution to getting the pc build working on my computer, and it ended up getting committed and shoved into this PR when it shouldn't have. once build is dockerized, it shouldn't be an issue.